src/Navbar.jsx

```diff
@@ -37,9 +66,7 @@ const Navbar = () => {
         <StyledListItem>
           <StyledLink to="/users/register">Register</StyledLink>
         </StyledListItem>
-        <StyledListItem>
-          <StyledLink to="/users/login">Login</StyledLink>
-        </StyledListItem>
+        <StyledListItem>{changeLoginStatus()}</StyledListItem>
```

Comment:
inline?
`{token ? <Button onClick={() => logout()}>Logout</Button> : <StyledLink to="/users/login">Login</StyledLink>}`